gen/generator.go

```diff
@@ -90,6 +91,7 @@ var templateFuncs = template.FuncMap{
 	"createNonExistingValue":       func(val string) string { return fmt.Sprintf("non_existing_%s", val) },
 	"getParentTestDependencies":    GetParentTestDependencies,
 	"getTargetTestDependencies":    GetTargetTestDependencies,
+	"setDefault":                   GetDefaultValues,
```

Comment:
Can we use `getDefaultValues` instead of `setDefault`?

Reply:
you're right, made the proper change in last commits. thanks.